README.rst

```diff
@@ -30,8 +30,8 @@ Features
 Requirements
 ------------
 
-django-timepiece is compatible with Django 1.6 (on Python 2.7 and Python 3.3),
-and Django 1.7 (on Python 2.7 and Python 3.4). PostgreSQL is the only
+django-timepiece is compatible with Django 1.7 (on Python 2.7 and Python 3.3),

```

Comment:
Django 1.7 support should be tested against Python 3.4. (Django 1.6 supported only up to Python 3.3.)